### PR TITLE
ling add paper

### PR DESCRIPTION
**Pull request by lanikaling**

### CODE REVIEW

**Comment by lanikaling** [comment]
Hi @sbillinge, the citation is added. Please review.

File: db/citations.yml [hunk 1]
```diff
@@ -7346,6 +7346,34 @@ liu;sr14:
...
+  grant: dmref19
+  journal: Journal of Applied Crystallography
+  month: feb
+  nb: ''
+  note: arXiv:2202.12335 [cond-mat.mtrl-sci]
+  number: ''
+  optannote: ''
+  optwwwlanl: ''
+  pages: ''
+  supplementary_info_urls: https://gitlab.thebillingegroup.com/software/20ll_sgcnnrobusttest
```
Comment (sbillinge) [inline db/citations.yml:7369]: this needs to be a list

File: db/citations.yml [hunk 2]
```diff
@@ -7346,6 +7346,34 @@ liu;sr14:
...
+  title: Robustness Test of the spacegroupMining Space Group Determination Model for
```
Comment (sbillinge) [inline db/citations.yml:7370]: we need tags and synopsis.  Look at Tim's paper for examples (not sure if it has them but if it does they will be a good start).  tags are for building reading lists based on the tags.  Synopsis is a short description of the main result that would be appropriate for a web-site, for example.

**Review by sbillinge — COMMENTED** [review]
nice job, couple of issues.

File: db/citations.yml [hunk 3]
```diff
@@ -7366,9 +7365,11 @@ llan;RobustnessTestSpacegroupMining22:
   optannote: ''
   optwwwlanl: ''
   pages: ''
-  supplementary_info_urls: https://gitlab.thebillingegroup.com/software/20ll_sgcnnrobusttest
-  title: Robustness Test of the spacegroupMining Space Group Determination Model for
-    atomic pair distribution functions
+  supplementary_info_urls: 
+    - https://gitlab.thebillingegroup.com/software/20ll_sgcnnrobusttest
+  synopsis: The spacegroupMining Space Group Determination Model is shown to be robust when applied to PDFs of different choices of experimental parameters rmax, Qmax, and atomic displacement parameters.
+  tags: robustness_test, machine_learning, data_mining, space_group, pair_distribution_function
```
Comment (sbillinge) [inline db/citations.yml:7371]: thanks. great.  remove pdf (everything is pdf!).  please check for machine learning and data mining if they are in the db with abbreviations

**Review by sbillinge — COMMENTED** [review]
nice, one comment

File: db/citations.yml (hunk 3 above)
Comment (lanikaling) [reply db/citations.yml]: Done.

**Pull request merged**